src/modules/ROOT/pages/changelog/202303/00.json

```
@@ -0,0 +1,800 @@
+{
+  "meta": {
+    "releasedOn": "2023.04.20"
+  },
+
+  "important": [
+    {
+      "what": [
+      "namespace import/export has some edge cases which prevent it from being exported.",
+      "Namely, one of them, is when you reference a deleted module from a page, that was crashes it."
```

```suggestion
      "Namely, when you reference a deleted module from a page and try to export the namespace."
```

One of them? What are the others? If there are, list them, else re-write it